PlusContent/compartment-mode.md

```diff
@@ -1 +1,19 @@
-Sandboxie Plus offers a less restrictive box mode, called Application Compartment. This mode provides improved application compatibility at the cost of reduced isolation.
\ No newline at end of file
+# Compartment Mode
+
+**NOTE: This feature requires a [supporter certificate](https://sandboxie-plus.com/supporter-certificate/).**
+
+The concept of an "Application Compartment" mode was introduced in **Sandboxie Plus v1.0.0**. This mode disables the normally used token-based security isolation in order to significantly improve compatibility while still retaining a level of security comparable to that of other available sandboxing products. It avoids many of the typical Sandboxie issues caused by processes running with a heavily restricted token.
+
+The setting for a compartment box can be enabled by adding `NoSecurityIsolation=y` to the box settings section of **Sandboxie.ini**. It can also be enabled in the Sandman UI. Right-click on a box and select "Sandbox Options" from the drop-down menu (or simply double-click on a box) to bring up the Box Options UI. Select the Box Type Preset as "Application Compartment (NO Isolation)" (with a **green** box icon) and click OK to apply changes. The status column of Sandman UI labels this box as **Application Compartment**.
+
+![](../Media/Box_CompartmentMode.png)
+
+In compartment mode, file system and registry filtering are still in place to enforce any access rules. So, processes do run without administrative privileges. This filtering can be disabled by adding `NoSecurityFiltering=y` to the box settings section of **Sandboxie.ini** in order to provide a greater degree of compatibility.
+
+To ensure that the security of this mode compares well with other sandboxing products, a new object access filter was developed and can be enabled by adding `EnableObjectFiltering=y` to the [GlobalSettings] section of **Sandboxie.ini**.
```

Comment:
`EnableObjectFiltering=y`  is enabled by default for new installations as of version [1.0.16 / 5.55.16](https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.16), this detail should be mentioned too:

> To facilitate optimal process isolation, the EnableObjectFiltering option is now on by default, although this only applies for new installations, hence it is recommended for existing installations to go to settings->advanced and enable it explicitly.

> EnableObjectFiltering is now set enabled by default, and replaces Sbie's old process/thread handle filter

Reply:
Agreed. Would appreciate your suggestions.

Reply:
Please let me know if the following text works for you:

![EnableObjectFiltering explanation](https://github.com/sandboxie-plus/sandboxie-docs/assets/12372772/67fcbd85-d411-446c-a082-b2391f7b2aa2)

Reply:
Please let me know if the following text works for you:

A new object access filter, enabled by default for new installations since **Sandboxie Plus v1.0.16**, replaces the Sandboxie's old process/thread handle filter to facilitate process isolation. For previous versions starting with **Sandboxie Plus v1.0.0**, it can be enabled by adding `EnableObjectFiltering=y` to the [GlobalSettings] section of **Sandboxie.ini**.

Reply:
Absolutely!